.gitignore

```diff
@@ -20,3 +20,7 @@
 # MATLAB
 *.asv
 *.m~
+src/examples/L-Room-rotation-multiPAAs/Input/node0paa.dat
+src/examples/L-Room-rotation-multiPAAs/Input/node1paa.dat
+src/examples/L-Room-rotation-multiPAAs/Output/Visualizer/NodePositions.json
+src/examples/SpatialSharing/Output/Visualizer/RoomCoordinates/RoomCoordinates.csv
```

Comment:
why?

Reply:
did I add this?

Reply:
I dont have files corresponding to line 23 24 and 26

Reply:
check now. I updated @mattia-lecci